globaldog.py

```diff
@@ -27,6 +27,8 @@ def handleErrors(datadogResponse):
     "app_key": os.environ.get("DATADOG_APP_KEY"),
```

Comment:
```python
- print(datadogCredentials)
```

The direct printing of credentials is a security risk. Consider using logging with proper data masking or removing this debug output entirely.